src/qml/patchmanager.h

```diff
@@ -68,7 +68,7 @@ class PatchManager: public QObject
     Q_PROPERTY(bool applyOnBoot READ applyOnBoot WRITE setApplyOnBoot NOTIFY applyOnBootChanged)
     Q_PROPERTY(bool notifyOnSuccess READ notifyOnSuccess WRITE setNotifyOnSuccess NOTIFY notifyOnSuccessChanged)
     Q_PROPERTY(bool bitnessMangle READ bitnessMangle WRITE setBitnessMangle NOTIFY bitnessMangleChanged)
-    Q_PROPERTY(QStringList mangleCandidates READ mangleCandidates)
+    Q_PROPERTY(QStringList mangleCandidates READ mangleCandidates NOTIFY mangleCandidatesChanged)
```

Comment:
Is this used now?

Reply:
It was "used" in that it is part of an expression, which led to:

`[W] unknown:0 - QQmlExpression: Expression file:///usr/lib64/qt5/qml/org/SfietKonstantin/patchmanager/SettingsPage.qml:125:23 depends on non-NOTIFYable properties:`